src/easypost.js

```diff
@@ -29,6 +31,8 @@ import TrackerService from './services/tracker_service';
 import UserService from './services/user_service';
 import WebhookService from './services/webhook_service';
 
+const util = require('util');
```

Comment:
This should not be exported/imported the way it is, I'll fix this in the modernization task

Reply:
We do need to have this import statement in order to use the `util` function: https://www.geeksforgeeks.org/node-js-util-format-method

I will leave this as it is now